.travis.yml

```diff
@@ -1,9 +1,18 @@
 sudo: false
 language: scala
-scala:
-- 2.11.7
 jdk:
-- oraclejdk8
+  - openjdk8
```

Comment:
Just delete the travis file, we don't use it

Reply:
Is Travis file is required for the open-source library it's for the Travis builds 
https://travis-ci.org/hmrc/play-language/builds/596028100?utm_source=github_status&utm_medium=notification

Reply:
@daniel-manning 
https://confluence.tools.tax.service.gov.uk/display/TEC/2019/07/02/Non-library+release+packages+will+be+deleted+from+Bintray
Travis build is still required for the library it's only not applicable to service in open, and play language is a library.

Reply:
It's not worth having an argument, I'll have someone else delete it later.